axum/src/serve.rs

```diff
@@ -15,7 +17,7 @@ use tower_service::Service;
 /// This method of running a service is intentionally simple and doesn't support any configuration.
 /// Use hyper or hyper-util if you need configuration.
```

Comment:
Do we need to update this comment as well?

Reply:
I wanted to keep it because we're still doing what _we_ believe to be sane defaults. I figured we'd leave this here to let the user know that they can configure custom stuff (TLS and what not) if they really want to

Reply:
Yes I think this comment should stay. We still won't expose any customization for this function even if we're wrapping hyper-util.